src/test/software/amazon/qldb/QldbDriverImplTest.java

```diff
@@ -141,72 +137,74 @@ public void testExecuteWithAvailableSession() throws IOException {
         assertTrue(returnedValue);
     }
 
-    /**
-     * When a session in the pool throws InvalidSessionException, the driver goes to the next session
-     * in the pool and executes the transaction
-     *
-     * @throws IOException
-     */
-    @Test
-    public void testExecutePicksAnotherSessionWhenStartTransactionFails() throws IOException {
-        mockClient.queueResponse(MockResponses.START_SESSION_RESPONSE);
-        mockClient.queueResponse(InvalidSessionException.builder().message("Msg1").build());
-
-        mockClient.queueResponse(MockResponses.START_SESSION_RESPONSE);
-        List<IonValue> parameters = Collections.emptyList();
-        queueTxnExecCommit(ionList, statement, parameters);
-
-
-        QldbDriver qldbDriverImpl = QldbDriver.builder()
-                                              .sessionClientBuilder(mockBuilder)
-                                              .ledger(LEDGER)
-                                              .maxConcurrentTransactions(2)
-                                              .build();
-        final Boolean result = qldbDriverImpl.execute(txn -> {
-            txn.execute(statement, Collections.emptyList());
-            return true;
-        });
-        assertTrue(mockClient.isQueueEmpty());
-        assertTrue(result);
-    }
-
-    @ParameterizedTest
-    @MethodSource("exceptionProvider")
-    public void testExecuteCustomPolicy(SdkException exception) {
-        RetryPolicy driverRetryPolicy = spy(RetryPolicy.maxRetries(3));
-        qldbDriverImpl = QldbDriver.builder()
-                                   .sessionClientBuilder(mockBuilder)
-                                   .ledger(LEDGER)
-                                   .maxConcurrentTransactions(POOL_LIMIT)
-                                   .transactionRetryPolicy(driverRetryPolicy)
-                                   .build();
-
-        mockClient.queueResponse(MockResponses.START_SESSION_RESPONSE);
-        mockClient.queueResponse(MockResponses.startTxnResponse("id"));
-        mockClient.queueResponse(exception);
-        mockClient.queueResponse(MockResponses.ABORT_RESPONSE);
-
-        RetryPolicy customRetryPolicy = spy(RetryPolicy.none());
-        assertThrows(exception.getClass(),
-                     () -> {
-             final Boolean result = qldbDriverImpl.execute(txn -> {
-                 txn.execute(statement, Collections.emptyList());
-                 return true;
-             }, customRetryPolicy);
-         });
-
-        verify(driverRetryPolicy, never()).maxRetries();
-        verify(customRetryPolicy, times(1)).maxRetries();
-    }
+//    /**
```

Comment:
Maybe add a todo about the commented tests stating why they are commented and when to uncomment them?